llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -1142,7 +1142,7 @@ adjustInstrProfile(std::unique_ptr<WriterContext> &WC,
     }
 
     // This name should have a static linkage.
-    size_t PostfixPos = NewName.find(FunctionSamples::UniqSuffix);
+    size_t PostfixPos = NewName.find(NameParticles::UniqSuffix);
```

Comment:
Can we use `StringRef::contains()` here too? Same with above, but I suspect `toString()` returns `std::string`.